frontend/javascripts/oxalis/controller/combinations/skeleton_handlers.ts

```diff
@@ -177,6 +177,17 @@ export function moveNode(dx: number, dy: number, nodeId?: number | null | undefi
     }),
   );
 }
+
+export function finishNodeMovement(nodeId: number) {
+  getSkeletonTracing(Store.getState().tracing).map((skeletonTracing) =>
+    getNodeAndTree(skeletonTracing, nodeId).map(([activeTree, node]) => {
+      Store.dispatch(
+        setNodePositionAction(V3.round(node.position, [0, 0, 0]), node.id, activeTree.treeId),
```

Comment:
I think, `round` makes most sense here. Note that the second parameter is the output variable and is necessary, because the mjs module would create a typed array otherwise (which immediately crashed during the freeze operation of the reducer). Probably, the TypedArray default of the mjs could use some re-thinking, too..